nuntium/tests/outbound_message_test.py

```diff
@@ -98,6 +100,23 @@ def test_successful_resend(self):
         self.assertEquals(outbound_message.status, "sent")
 
 
+    def test_message_resend_form(self):
+        outbound_message = OutboundMessage.objects.create(
+            message=self.message,
+            contact=self.contact1,
+            site=Site.objects.get_current()
+        )
+        outbound_message.send()
+
+        response = self.client.post(reverse('messages_per_writeitinstance', subdomain=WriteItInstance.objects.get(id=1).slug), data={
```

Comment:
Do an assertContains which by default checks that the status code was 200. https://docs.djangoproject.com/en/1.8/topics/testing/tools/#django.test.SimpleTestCase.assertContains

If messages is working, you can check that your feedback message was in the response  as well. Otherwise just check for something vaguely relevant like the page title.